frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.js

```diff
@@ -94,6 +94,10 @@ export default function NextStepsRepeater({
 
   const stepType = name === 'specialistNextSteps' ? 'specialist' : 'recipient';
 
+  const dateLabel = (index) => (stepType === 'recipient'
```

Comment:
Does this need to change for other entity?

Reply:
[Fixed](https://github.com/HHS/Head-Start-TTADP/pull/1080/commits/19e002a4f15fc8860894a87fa771f030fae311be)